src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyTestDefault.java

```diff
@@ -0,0 +1,26 @@
+/*
+RightCurly
+option = (default)SAME
+tokens = (default)LITERAL_TRY , LITERAL_CATCH , LITERAL_FINALLY , LITERAL_IF , LITERAL_ELSE
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.blocks.rightcurly;
+
+public class InputRightCurlyTestDefault {
```

Comment:
@arinmodi do we require two inputs? If not, let's have only one please.

Reply:
Both the input file having different configuration that's why

Reply:
Mutation was killed with a single config in https://github.com/checkstyle/checkstyle/commit/e5b77a6fe9c2724b2215f4cadca5bbda7270aacd, why do we need two now?

Reply:
You told me to add this type of input in previous item

Now, Are you sure want to remove one input file ?

Reply:
Input code and config we check it with are different concepts; please consolidate all input to one file with config to kill mutation.